tests/auto/make_tests.py

```diff
@@ -16,6 +16,8 @@
         cnt = op_count.get(test['mnemonic'],0)
         if cnt > 1000: #No more than 1000 instructions of each kind
```

Comment:
Can you also momentarily tweak this counter to generate ~10? tests for each instruction and send them with the PR.
Add a file like tests/test_x86pcmpx.py  or another name you prefer.

Reply:
Sure, just pushed the full test cases, there's 64 test cases for each instruction. And I also had one more test case for memory type operand ("pcmpistri $0b00011011, %q1, %%xmm1\n").